fortitude/src/check.rs

```diff
@@ -818,6 +994,54 @@ pub struct RuleSelection {
     pub select: Option<Vec<RuleSelector>>,
     pub ignore: Vec<RuleSelector>,
     pub extend_select: Vec<RuleSelector>,
+    pub fixable: Option<Vec<RuleSelector>>,
+    pub unfixable: Vec<RuleSelector>,
+    pub extend_fixable: Vec<RuleSelector>,
```

Comment:
We haven't implemented these options. Is that something we should get done before the next release?

Reply:
I think that would be good but not essential. I added them here to be able to just use the whole section from ruff